integration/test/ParseUserTest.js

```diff
@@ -986,7 +986,7 @@ describe('Parse User', () => {
     expect(Parse.AnonymousUtils.isLinked(user)).toBe(true);
   });
 
-  it('can link with twitter', async () => {
+  fit('can link with twitter', async () => {
```

Comment:
Should we `xit` this test. These tests doesn't exist on the server for a reason.